### ISSUE TITLE
[Accessibility]: Tooltip appearing on hover over the HistogramChart is not keyboard operable

### ISSUE THREAD

**Issue by vijaysince89** [carbon-design-system/carbon-charts#1920]
### Application/Team

Insights (GDSC)

### Environment

Assistive technology used to verify, Browser

### What happened? What did you expect to happen instead?

When we hover over the charts we see there are some tooltip appearing on the HistogramChart. We could not able to access these tooltip using the keyboard (tabs). Because of this the assistive technology that reads the screen text is not able to read these toot tips.

### What WCAG 2.1 checkpoint does the issue violate?

Content exposed via hover should be keyboard operable

### Version

"@carbon/charts-react": "^1.15.6"

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P3
